veracruz-mcu-client/prj.conf

```diff
@@ -0,0 +1,69 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+#CONFIG_MAIN_STACK_SIZE=16384
+#CONFIG_MAIN_STACK_SIZE=8192
+CONFIG_MAIN_STACK_SIZE=16384
+CONFIG_SIZE_OPTIMIZATIONS=n
+
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_TCP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+
+##CONFIG_NET_L2_ETHERNET=y
+##CONFIG_NET_QEMU_ETHERNET=y
```

Comment:
Why are some of the configuration options in this file commented out?  Are they stray comments, or do we need to keep them?

Reply:
It was just a part of debugging, the commented configurations may be useful if someone attempts to resurrect the qemu-lm4f port, but I don't think we plan to